people-details-sql-jdbc/src/main/java/menu/ListAllPerson.java

```diff
@@ -0,0 +1,16 @@
+package menu;
+
+import connector.SQLConnector;
+import loader.PropertiesLoader;
+
+public class ListAllPerson {
```

Comment:
I mean this class sets up the connection, doesn't it? If so, I think we could find a better name ;)